### DISCUSSION TITLE
Support for DirectX 9 Games

### DISCUSSION THREAD

**Discussion by arthrfrts** [MythicApp/Mythic#145] [General]
Hello!

I'm trying to install DirectX 9 games on macOS for a while now, and would like to try using Mythic and see if I get better results. Usually, I've been getting better performance and feature support from [Gcenx DXVK/D9VK and WineskinServer](https://github.com/gcenx/WineskinServer).

I've seen that in [this commit](https://github.com/MythicApp/Engine/commit/af22b1073aeaa5c4b6a494f43e10813e74769c08) you removed the DX9-related DLLs from DXVK. Is there any support for DirectX 9 games for now, or is it only WineD3D? — which, in my tests, performed poorly and never managed to run The Sims 2.

**Comment by JoshBashed** [comment]
They are not needed because wine has built-in dx9 support.

**Reply by Meaticus22** [reply]
What about DXMT, how does that work?
